### PR TITLE
Correct error checkstyle and implement account api

### PR DESCRIPTION
**Pull request by oscarlopez8**

### CODE REVIEW

File: src/main/java/salesforce/Util.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
+package salesforce;
+
+/**
+ * Class.
```
Comment (LimbertVargas) [inline src/main/java/salesforce/Util.java:16]: please add more description for this documentation.

File: src/main/java/salesforce/api/AccountAPI.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+import io.restassured.response.Response;
+import salesforce.api.rest.RestClientAPI;
+
+import java.util.Map;
+
+import static salesforce.api.EndPoints.ACCOUNT_ENDPOINT;
+import static salesforce.api.EndPoints.SLASH;
+
+/**
+ * AccountAPI class.
```
Comment (LimbertVargas) [inline src/main/java/salesforce/api/AccountAPI.java]: Improve this class documentation.

File: src/main/java/salesforce/pages/LoginPage.java [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+    /**
+     * Constant for the title of home page.
+     */
+    public static final String HOME_PAGE_TITLE = "Home Page ~ Salesforce - Developer Edition";
+
+    /**
+     * Web element for the user name text-box.
+     */
+    @FindBy(css = "input#username")
```
Comment (LimbertVargas) [inline src/main/java/salesforce/pages/LoginPage.java:36]: Why do you use css locator for this web element?, maybe you could maintain this locator as "id=username"

File: src/main/java/salesforce/pages/LoginPage.java [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+    @FindBy(id = "password")
+    private WebElement passwordTextBox;
+
+    /**
+     * Web element for the login button.
+     */
+    @FindBy(id = "Login")
+    private WebElement loginButton;
+
+
```
Comment (LimbertVargas) [inline src/main/java/salesforce/pages/LoginPage.java]: please delete this extra line

File: src/main/java/salesforce/pages/TransporterPage.java [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+ */
+
+package salesforce.pages;
+
+import core.selenium.WebDriverManager;
+import core.selenium.util.SalesForceGetProperties;
+import org.openqa.selenium.WebDriver;
+
+/**
+ * TransporterPage class.
```
Comment (LimbertVargas) [inline src/main/java/salesforce/pages/TransporterPage.java]: Please, improve this class documentation.

**Review by LimbertVargas — COMMENTED** [review]
please fix this observations

File: src/main/java/salesforce/pages/LoginPage.java (hunk 3 above)
Comment (oscarlopez8) [reply src/main/java/salesforce/pages/LoginPage.java:36]: I tested if it works with that kind of CSS locator

Comment (oscarlopez8) [reply src/main/java/salesforce/pages/LoginPage.java:36]: And works

**Review by LimbertVargas — APPROVED** [review]

**Pull request merged**